### PR TITLE
CY-539-fix-blueprint-tar-permissions

### PR DESCRIPTION
**Pull request by idob-cloudify**

### CODE REVIEW

File: cloudify_rest_client/utils.py [hunk 1]
```diff
@@ -28,10 +28,14 @@ def tar_file(file_to_tar, destination_dir, tar_name=''):
     :param tar_name: optional tar name.
     :return:
     """
+    def _reset_tarinfo(tarinfo):
+        tarinfo.mode = 0755
```
Comment (tehasdf) [inline cloudify_rest_client/utils.py]: might as well use `0o755` syntax for py2+3-compat

Comment (isaac-s) [inline cloudify_rest_client/utils.py]: - What about file ownership? can / should we guarantee `cfyuser:cfyuser`? Maybe `cfyuser:adm`?
- Not sure that `0755` is the right thing to do (depends on the file ownership we choose to go with)

Comment (tehasdf) [reply cloudify_rest_client/utils.py]: it is going to be cfyuser guaranteed, because that's the user that saves the file (restservice)

Comment (isaac-s) [reply cloudify_rest_client/utils.py]: @tehasdf and the group? will be the primary group of `cfyuser`, right?

Comment (tehasdf) [reply cloudify_rest_client/utils.py]: i believe so

**Comment by tehasdf** [comment]
@isaac-s I've dismissed your review, because this PR is no longer setting 0755, it's now setting o+r

**Pull request merged**